test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -7,6 +7,7 @@
 import java.util.List;
 
 import org.junit.Test;
+import static org.junit.Assert.*;
```

Comment:
Import classes with the exact name instead of using *. This is a coding standard violation.